relay-event-schema/src/protocol/span.rs

```diff
@@ -290,10 +290,10 @@ pub struct SpanData {
     pub user: Annotated<Value>,
 
     /// Replay ID
-    #[metastructure(field = "replay_id")]
+    #[metastructure(field = "sentry.replay.id", legacy_alias = "replay_id")]
     pub replay_id: Annotated<Value>,
 
-    #[metastructure(field = "sdk.name")]
+    #[metastructure(field = "sentry.sdk.name")]
```

Comment:
No legacy alias need here, this field was only introduced in https://github.com/getsentry/relay/pull/3456.